bibliopixel/drivers/SPI/base.py

```diff
@@ -0,0 +1,27 @@
+from .. channel_order import ChannelOrder
+from .. driver_base import DriverBase
+from ... import gamma as _gamma
+from ... util.enum import resolve_enum
+from . import interfaces
+
+
+class SPIBase(DriverBase):
+    """Base driver for controling SPI devices on systems like the Raspberry Pi and BeagleBone"""
+    def __init__(self, num, dev='/dev/spidev0.0',
+                 interface='FILE', spi_speed=1,
```

Comment:
The SpiFileInterface is now really stupid (you can't set spi setting like speed), but it is fast.
You should use SpiPeripheryInterface or SpiPyDevInterface as default.